build.sh

```diff
@@ -2,7 +2,7 @@
 ## START STANDARD SITE BUILD SCRIPT INCLUDE
 readonly THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
 readonly BOOTSTRAP="$(dirname "$THIS_SCRIPT")/resources/bootstrap.inc.sh"
-readonly BOOTSTRAP_VERSION=v0.8
+readonly BOOTSTRAP_VERSION=v0.9
```

Comment:
I think this should let you remove the png's from this PR (along with image-randomizer.php)

```suggestion
readonly BOOTSTRAP_VERSION=chore/v0.10
```

Reply:
Note: we never merge BOOTSTRAP_VERSIONs that are not published into master. Once the v0.10 is published, we can merge this.

```suggestion
readonly BOOTSTRAP_VERSION=v0.10
```

Reply:
Hi @darcywong00, this PR is not ready yet. I am still dealing with the shared-site image retrieves.